src/api/api.js

```diff
@@ -157,6 +157,44 @@ function searchBeaconFreq(datasetId, start, end, referenceName) {
   });
 }
 
+/*
+Fetch variant set for a specific Dataset Id; and returns a promise
+ * @param {string}... Dataset ID
+*/
+function searchVariantSets(datasetId) {
+  return fetch(`${BASE_URL}/variantsets/search`, {
+    method: 'post',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      datasetId,
+    }),
+  }).then((response) => {
+    if (response.ok) {
+      return response.json();
+    }
+    return {};
+  });
+}
+
+/*
+Fetch reference set for a specific referenceSetId; and returns a promise
+ * @param {string}... Reference set ID
+*/
+function searchReferenceSets(referenceSetsId) {
```

Comment:
this is not correct, to get the info of a referenceset, make a GET request to `/referencesets/{referenceSetsId}`, not a POST request, the POST request will always return a list of referencesets on the server